### PR TITLE
ARR: Post emergency score to new invitation

### PR DESCRIPTION
**Pull request by haroldrubio**

### CODE REVIEW

File: openreview/arr/process/emergency_load_process.py [hunk 1]
```diff
@@ -170,29 +153,27 @@ def process(client, edit, invitation):
...
+    for edge in aggregate_score_edges:
+        edges_to_post.append(openreview.api.Edge(
+            invitation=f"{role}/-/Emergency_Score",
+            readers=edge.readers,
+            writers=edge.writers,
+            nonreaders=edge.nonreaders,
+            signatures=edge.signatures,
+            head=edge.head,
+            tail=edge.tail,
+            weight=edge.weight + 10
```
Comment (melisabok) [inline openreview/arr/process/emergency_load_process.py:166]: is there a reason to sum 10 here?

**Comment by melisabok** [comment]
Fixes https://github.com/openreview/openreview-py/issues/2127

**Comment by melisabok** [comment]
@haroldrubio the tests are failing here because they expect to be higher than 10, can you fix that?

File: openreview/arr/helpers.py [hunk 2]
```diff
@@ -1034,6 +1035,25 @@ def __init__(self, client_v2, venue, configuration_note, request_form_id, suppor
                     invitation=stat_inv
                 )
 
+            if not openreview.tools.get_invitation(self.client_v2, f"{role}/-/Emergency_Score"): # Hold "Requested" or "Reassigned", head=submission ID
+                m._create_edge_invitation(f"{role}/-/Emergency_Score")
```
Comment (melisabok) [inline openreview/arr/helpers.py:1039]: @haroldrubio do we need to add the Emergency_Score to the edge browser urls?

Comment (haroldrubio) [reply openreview/arr/helpers.py:1039]: Yes I'll be changing the links in my webfield branch

Comment (haroldrubio) [reply openreview/arr/helpers.py:1039]: Oh I see I'll update the links in my branch after this is merged

**Review by melisabok — APPROVED** [review]

**Pull request merged**